src/SessionCipher.js

```diff
@@ -287,6 +287,10 @@ SessionCipher.prototype = {
           return Promise.resolve(); // Already calculated
       }
 
+      if (counter - chain.chainKey.counter > 2000) {
+          throw new Error('Over 2000 messages into the future!');
```

Comment:
Nit: Not sure what the lint setup is for this project but the other `Error`s seem to use double quotes for strings.